packages/core/src/components/essentials/legend.ts

```diff
@@ -227,6 +268,15 @@ export class Legend extends Component {
 		const self = this;
 		const svg = this.getContainerSVG();
 		const options = this.model.getOptions();
+		const legendOptions = Tools.getProperty(options, "legend");
+		let truncationThreshold = Tools.getProperty(
+			legendOptions,
+			"truncation",
+			"threshold"
+		);
+		if (!truncationThreshold) {
+			truncationThreshold = Configuration.legend.truncation.threshold;
+		}
```

Comment:
legend options that you've set defaults for in configuration.ts are part of the base chart options which are used in merging user-provided options with our defaults

so your default legend values will be there unless the user sets them to null or undefined, you don't need to set them manually

Reply:
Resolved. Thanks.